src/utilities/utilities.js

```diff
@@ -40,6 +40,33 @@ export function round(number, precision) {
  * @typedef {"ankle_draft" | "ashrae" | "iso" | "ISO7933"} Standard
  */
 
+function degrees_to_radians(degrees) {
+  var pi = Math.PI;
+  return degrees * (pi / 180);
```

Comment:
just use `Math.PI` in the calculation, no need for a variable. Same in the other function